packages/intrepid2/README.md

```diff
@@ -0,0 +1,12 @@
+# Intrepid2 Package
+
+
+## Copyright and License
+See intrepid2/COPYRIGHT, intrepid2/LICENSE, https://trilinos.github.io/license.html and individual file headers for additional information.
+
+
+## Questions? 
+Contact lead developers:
+
+* Intrepid2 team     (GitHub handle: @trilinos/intrepid2)
+* Mauro Perego       (GitHub handle: [mperego](https://github.com/mperego) or mperego@sandia.gov)
```

Comment:
Can you add Nathan Roberts as well? 

* Nathan Roberts (GitHub handle: [CamelliaDPG](https://github.com/CamelliaDPG) or nvrober@sandia.gov)

Reply:
Nathan V. Roberts